packages/lab/src/slider/Slider.css

```diff
@@ -1,201 +1,109 @@
-.salt-density-high {
-  --slider-clickable-paddingTop: var(--saltSlider-clickable-paddingTop, 2px);
-  --slider-clickable-paddingBottom: var(--saltSlider-clickable-paddingBottom, 2px);
-  --slider-paddingTop: var(--saltSlider-paddingTop, 3px);
-  --slider-paddingBottom: var(--saltSlider-paddingBottom, 0);
-}
-
-.salt-density-medium {
-  --slider-clickable-paddingTop: var(--saltSlider-clickable-paddingTop, 4px);
-  --slider-clickable-paddingBottom: var(--saltSlider-clickable-paddingBottom, 4px);
-  --slider-paddingTop: var(--saltSlider-paddingTop, 6px);
-  --slider-paddingBottom: var(--saltSlider-paddingBottom, 0);
+.saltSlider {
+  display: grid !important;
+  gap: var(--salt-spacing-100);
+  grid-template-columns: auto 1fr auto;
 }
 
-.salt-density-low {
-  --slider-clickable-paddingTop: var(--saltSlider-clickable-paddingTop, 8px);
-  --slider-clickable-paddingBottom: var(--saltSlider-clickable-paddingBottom, 8px);
-  --slider-paddingTop: var(--saltSlider-paddingTop, 9px);
-  --slider-paddingBottom: var(--saltSlider-paddingBottom, 0);
+.saltSlider-bottomLabel {
+  grid-template-columns: 1fr;
```

Comment:
only a single grid column when the labels are on the bottom - slider to all the available space